docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -118,26 +107,20 @@ it's more than that -- you can also read variables about other hosts.  We'll sho
 
 .. seealso::
 
-    :doc:`playbooks_templating`
+    :ref:`playbooks_templating`
         More information about Jinja2 templating
 
 .. _jinja2_filters:
 
-Jinja2 Filters
-``````````````
-
-.. note:: These are infrequently utilized features.  Use them if they fit a use case you have, but this is optional knowledge.
+Transforming variables: Jinja2 filters
+======================================
 
-Filters in Jinja2 are a way of transforming template expressions from one kind of data into another.  Jinja2
-ships with many of these. See `builtin filters`_ in the official Jinja2 template documentation.
-
-In addition to those, Ansible supplies many more. See the :doc:`playbooks_filters` document
-for a list of available filters and example usage guide.
+Jinja2 filters let you transform the value of a variable within a template expression. For example, the ``capitalize(s)`` filter capitalizes any value passed to it; the ``to_yaml`` and ``to_json`` filters change the format of your variable values. Jinja2 includes many `built-in filters <http://jinja.pocoo.org/docs/templates/#builtin-filters>`_ and Ansible supplies :ref:`many more filters <playbooks_filters>`.
```

Comment:
Why do you write `capitalize(s)` (with argument), but not `to_yaml(data)` or `to_json(data)`? I would leave the `(s)` away

Reply:
good point @felixfontein, I'll fix that